siriuspy/siriuspy/servconf/types/rmp_bo_ps.py

```diff
@@ -1,10 +1,10 @@
 """Booster ramp configuration definition."""
 import copy as _copy
-from siriuspy.csdevice.pwrsupply import default_wfmsize as \
-    _default_wfmsize
+from siriuspy.csdevice.pwrsupply import max_wfmsize as \
+    _max_wfmsize
```

Comment:
The same import in all the other files has only one line. Why use 2 lines here?

Reply:
it had to be a two-liner when the imported variable was named '_default_wfmsize'.  text auto replace in atom was not that smart! :smiley:. changing it...